runbotics/runbotics-desktop/src/core/websocket/bpmn/runtime-subscriptions.service.ts

```diff
@@ -16,15 +16,18 @@ import dayjs from 'dayjs';
 export class RuntimeSubscriptionsService {
     constructor(
         private readonly runtimeService: RuntimeService,
-        @InjectIoClientProvider() private readonly io: IoClient,
+        @InjectIoClientProvider() private readonly io: IoClient
     ) {}
 
-    private readonly logger = new RunboticsLogger(RuntimeSubscriptionsService.name);
+    private readonly logger = new RunboticsLogger(
+        RuntimeSubscriptionsService.name
```

Comment:
leave these indentations and trailing commas alone xd

Reply:
no.